src/code/z_bg_collect.c

```diff
@@ -90,12 +90,12 @@ u32 BgCheck2_UpdateActorAttachedToMesh(CollisionContext* colCtx, s32 index, Acto
         return 0;
     }
 
-    if ((meshActor->unk154 & 1) != 0) {
+    if ((meshActor->flags & 1) != 0) {
```

Comment:
For bitwise checks like this you can drop the `!= 0` 
```suggestion
    if (meshActor->flags & 1) {
```

Reply:
Technically out of scope as the only changes to this file were automated. Updated anyway. z_bg_collect will need a documentation pass eventually.